Sources/ScheduledTasks.php

```diff
@@ -23,6 +23,16 @@ function AutoTask()
 {
 	global $smcFunc;
 
+	// We bail out of index.php too early for these to be called.
+	frameOptionsHeader();
+	corsPolicyHeader();
+
+	if (isset($_SERVER['HTTP_X_SMF_AJAX']) && isset($_SERVER['REQUEST_METHOD']) && $_SERVER['REQUEST_METHOD']	=== 'OPTIONS')
+	{
+		send_http_status(204);
+		die;
```

Comment:
obExit() seems lonely

Reply:
We don't need to call it in obExit because we do it in smf_main, just as our frame security does.